src/DOM.js

```diff
@@ -175,6 +175,9 @@ angular.module('textAngular.DOM', ['textAngular.factories'])
 								}
 							}
 						}
+						_nodes = _nodes.filter(function(value, index, self) {
+							return self.indexOf(value) === index;
+						});
```

Comment:
The problem has been in a multiple of child inline elements with selected block element. 
All child inline elements needs a block wrapper. My fix prevents duplication of created wrappers. 
I think - it may be more correct decision. But I opted this solution for its simplicity and clarity.